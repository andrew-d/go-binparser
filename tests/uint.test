[
    {"name": "UintTest",
     "fields": [
        {"name": "f_uint8", "offset": 0, "type": "uint8"},
        {"name": "f_uint16", "offset": 1, "type": "uint16"},
        {"name": "f_uint32", "offset": 3, "type": "uint32"},
        {"name": "f_uint64", "offset": 7, "type": "uint64"}
    ]}
]
