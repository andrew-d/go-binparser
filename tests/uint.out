F_Uint8,%!s(uint8=18)
F_Uint16,%!s(uint16=22068)
F_Uint32,%!s(uint32=3450572920)
F_Uint64,%!s(uint64=8603657889541919215)
